clientContext.ts

```diff
@@ -3,17 +3,20 @@ import React, {Context} from 'react';
 export interface ClientProps {
   nationalAvalancheCenterHost: string;
   snowboundHost: string;
+  nwacHost: string;
```

Comment:
very minor: I'm using `observationsUrlPrefix` instead of `Host` for the obs path, since these technically aren't hosts.

Reply:
`nwac.us` is a host, what do you mean re: prefixing?

Reply:
I mean that `https://nwac.us` is not a host, it's a url prefix 😄

Reply:
A level of hair-splitting I don't really have strong opinions over :upside_down_face: